### PR TITLE
some client names and moved PageTurnWidget

### PR DESCRIPTION
**Pull request by shedaniel**

### CODE REVIEW

File: mappings/net/minecraft/client/gui/screen/recipebook/RecipeAlternatesWidget.mapping [hunk 1]
```diff
@@ -13,10 +13,13 @@ CLASS net/minecraft/class_508 net/minecraft/client/gui/screen/recipebook/RecipeA
 		ARG 1 mouseX
 		ARG 3 mouseY
 	METHOD method_2613 setVisible (Z)V
+		ARG 1 visible
 	METHOD method_2614 getResults ()Lnet/minecraft/class_516;
 	METHOD method_2615 getLastClickedRecipe ()Lnet/minecraft/class_1860;
 	METHOD method_2616 isVisible ()Z
 	METHOD method_2617 showAlternatesForResult (Lnet/minecraft/class_310;Lnet/minecraft/class_516;IIIIF)V
+		ARG 1 client
+		ARG 2 resultCollection
```
Comment (liach) [inline mappings/net/minecraft/client/gui/screen/recipebook/RecipeAlternatesWidget.mapping]:
```suggestion
		ARG 2 results
```

File: mappings/net/minecraft/client/gui/screen/recipebook/RecipeBookGhostSlots.mapping [hunk 2]
```diff
@@ -9,6 +9,7 @@ CLASS net/minecraft/class_505 net/minecraft/client/gui/screen/recipebook/RecipeB
 		ARG 1 client
 		ARG 2 x
 		ARG 3 y
+		ARG 4 isBig
```
Comment (liach) [inline mappings/net/minecraft/client/gui/screen/recipebook/RecipeBookGhostSlots.mapping:12]: Like the output slot or?

File: mappings/net/minecraft/client/gui/screen/recipebook/RecipeBookWidget.mapping [hunk 3]
```diff
@@ -32,6 +32,7 @@ CLASS net/minecraft/class_507 net/minecraft/client/gui/screen/recipebook/RecipeB
 	METHOD method_2581 drawGhostSlots (IIZF)V
 		ARG 1 left
 		ARG 2 top
+		ARG 3 isBig
```
Comment (liach) [inline mappings/net/minecraft/client/gui/screen/recipebook/RecipeBookWidget.mapping:35]: Sounds more like if it's in block screen or player inventory screen.

File: mappings/net/minecraft/client/gui/screen/recipebook/RecipeBookGhostSlots.mapping (hunk 2 above)
Comment (shedaniel) [reply mappings/net/minecraft/client/gui/screen/recipebook/RecipeBookGhostSlots.mapping:12]: yes, it is to draw the ghost slot background (the red overlay) bigger, but it doesn't have to be an output slot.

File: mappings/net/minecraft/client/gui/screen/recipebook/RecipeBookWidget.mapping (hunk 3 above)
Comment (shedaniel) [reply mappings/net/minecraft/client/gui/screen/recipebook/RecipeBookWidget.mapping:35]: this is same as the bigger ghost slot comment made above.

**Review by liach — APPROVED** [review]

**Review by Prospector — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**